lib/cache/repositories/impressions/redis_repository.rb

```diff
@@ -39,14 +39,15 @@ def add_bulk(key, bucketing_key, treatments_labels_change_numbers, time)
           # delete fetched impressions afterwards
           def clear
             impressions = impression_keys.each_with_object([]) do |key, memo|
-              _, _, ip, = key.split('/')
+              _, _, ip, _ = key.split('/')
```

Comment:
too much magic and not very readable.

Mind adding a comment indicating a sample of the input so it's easier for reviews to see what's going on?